packages/nx-plugin/src/generators/service/files/__dot__dependency-cruiser.js__tmpl__

```diff
@@ -0,0 +1,5 @@
+const commonDependencyCruiserConfig = require('<%= offsetFromRoot %>/commonConfiguration/dependency-cruiser.config');
+
+const path = ['functions'];
```

Comment:
Why only `functions`? Shouldn't we include everything except the `node_modules` ?